lib/Drush/CommandFiles/core/DrupliconCommands.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace Drush\CommandFiles\core;
+
+class DrupliconCommands {
+
+  /*
+   * @todo not working until we modernize drush_get_global_options().
+   * @hook alter help
+   */
+  public function help($command) {
+    if ($command['command'] == 'global-options' && $command['#brief'] === FALSE) {
+      $command['options']['druplicon'] = [
+        'description' => 'Shows the druplicon as glorious ASCII art.',
+      ];
+    }
+  }
+
+  /**
+   * Print druplicon as post-command output.
+   *
+   * @hook post-command
```

Comment:
post-command currently doesn't do what one might expect -- but never mind about that, I can and will fix it.  (If you run this today, though, you'll probably see post-command run before the main command hook.)